### ISSUE TITLE
Add Retweet feature 

### ISSUE THREAD

**Issue by whythoSanskar** [whythoSanskar/TwitterBot#5]
As of now the bot can only tweet messages and send some images 
- Add a retweet feature to the bot so that it can retweet from other tweets

**Comment by yashgosa** [comment]
1. Go through the readme 
2. Try install the required packages listed in the `requirements.txt`
3. If it says `requirements.txt not found` that make sure you have cloned this repo on your machine and you are in the `TwitterBot\Twitter_Bot\Environment` directory.

**Comment by yashgosa** [comment]
@whythoSanskar  `Import "keys_test" could not be resolved` what's `key tests` its seems it wasnt there in `requirements.txt` and in the provided code

**Comment by whythoSanskar** [comment]
Hey @sofezticated-gentleman, well i just created that file initially to store my API keys and secret. Well that is something I need to change it too.

**Comment by whythoSanskar** [comment]
> 1. Go through the readme 
> 2. Try install the required packages listed in the `requirements.txt`
> 3. If it says `requirements.txt not found` that make sure you have cloned this repo on your machine and you are in the `TwitterBot\Twitter_Bot\Environment` directory.

Yeah I should append this to the readme file and i have to add tweepy documentation link too. I need to change and be more specific in readme

**Comment by yashgosa** [comment]
> > 1. Go through the readme
> > 2. Try install the required packages listed in the `requirements.txt`
> > 3. If it says `requirements.txt not found` that make sure you have cloned this repo on your machine and you are in the `TwitterBot\Twitter_Bot\Environment` directory.
> 
> Yeah I should append this to the readme file and i have to add tweepy documentation link too. I need to change and be more specific in readme

@whythoSanskar after doing so, assign this issue to me

**Comment by whythoSanskar** [comment]
Sure @sofezticated-gentleman, you can start working on this issue.
